oat-sa/tao-core-ui-fe
# fix/AUT-3688/mockup-mismatches

related to https://oat-sa.atlassian.net/browse/AUT-3688

### Description

Update of the font icons to address solar design differencies, minor styling fixes

### How to test

Check with tao-core part 

To check the list of icons you need to install extension [oat-sa/extension-tao-devtools](https://github.com/oat-sa/extension-tao-devtools) and open Tools -> Tao Icon Font in tao
